# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: uriloader/exthandler/nsExternalHelperAppService.h
# Commit: 68bc2ae9b89a
# Full Hash: 68bc2ae9b89a74996c48d8e885a3f06e85c679bc
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2019-11-12 09:43:07
# Regressor Bug: 1589270
# File Overlap Count: 1
# Description:
#   Bug 1589270 - Part 2: Remove support for handling the Refresh header while processing a download. r=bzbarsky
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D49527
# ==============================================================================

diff -r 73f846dfc5b1 -r 68bc2ae9b89a uriloader/exthandler/nsExternalHelperAppService.h
--- a/uriloader/exthandler/nsExternalHelperAppService.h	Mon Nov 11 21:02:46 2019 +0000
+++ b/uriloader/exthandler/nsExternalHelperAppService.h	Fri Nov 08 03:01:03 2019 +0000
@@ -402,15 +402,6 @@
   nsresult ContinueSave(nsIFile* aFile);
 
   /**
-   * After we're done prompting the user for any information, if the original
-   * channel had a refresh url associated with it (which might point to a
-   * "thank you for downloading" kind of page, then process that....It is safe
-   * to invoke this method multiple times. We'll clear mOriginalChannel after
-   * it's called and this ensures we won't call it again....
-   */
-  void ProcessAnyRefreshTags();
-
-  /**
    * Notify our nsITransfer object that we are done with the download.  This is
    * always called after the target file has been closed.
    *
@@ -450,8 +441,6 @@
    */
   nsCOMPtr<nsITransfer> mTransfer;
 
-  nsCOMPtr<nsIChannel> mOriginalChannel; /**< in the case of a redirect, this
-                                            will be the pre-redirect channel. */
   nsCOMPtr<nsIHelperAppLauncherDialog> mDialog;
 
   /**
